light-zk.js/src/wallet/user.ts

```diff
@@ -77,7 +72,6 @@ export class User {
   provider: Provider;
   account: Account;
   transactionHistory?: UserIndexedTransaction[];
-  private seed?: string;
```

Comment:
why remove this? is this unused?

Reply:
I think we have it in the account already